python_ta/reporters/node_printers.py

```diff
@@ -128,6 +128,19 @@ def render_missing_space_in_doctest(msg, _node, source_lines=None):
     yield from render_context(line + 1, line + 3, source_lines)
 
 
+def render_pep8_blank_lines_in_between_functions(msg, _node, source_lines=None):
+    """Render a PEP8 blank lines in between functions message."""
+    line = msg.line - 1
+    while source_lines[line - 1] == "\n":
+        line -= 1
+    yield from render_context(line - 1, line + 1, source_lines)
+    yield from (
+        (curr_line, slice(None, None), LineType.ERROR, " ")
```

Comment:
Based on the point about visibility that was raised in Monday's meeting, one simple idea is to put in a larger number of spaces here (e.g. 20) for rendering.